modules/serverless-kafka-broker-tls-custom-config.adoc

```diff
@@ -0,0 +1,75 @@
+// Module is included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kafka.adoc
+
+[id="serverless-kafka-broker-tls-custom-config_{context}"]
+= Configuring TLS authentication for the Kafka Broker
+
+.Prerequisites
+
+* A Kafka cluster CA certificate as a `.pem` file.
+* A Kafka cluster client certificate and key as `.pem` files.
+
```

Comment:
all the prerequisites for each module should probably have the same bit about installing serverless, eventing, etc.

Reply:
I copied the `Prerequisites` from the `modules/serverless-kafka-tls.adoc` file, which also does not list serverless, eventing etc as required.